### DISCUSSION TITLE
About CODEOWNERS - User with Write permissions also can setup?

### DISCUSSION THREAD

**Discussion by ystndr** [community/community#36497] [Repositories] [answered]
In [About code owners](https://docs.github.com/ja/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners), it's stated that `People with admin or owner permissions can set up a CODEOWNERS file in a repository`.

I have a repo where I got only a Write permission. 
In the main branch of that repo, the branch protection rule settings for `Require review from Code Owners` is enabled
<img width="781" alt="image" src="https://user-images.githubusercontent.com/31244806/196315886-0e014fcf-2d28-43ad-ac43-39bd1bacea97.png">

But I can still create a PR that creates a CODEOWNERS file there (of course still requiring approval to merge).

My question, does that mean people with Write permission can also create/modify/delete the CODEOWNERS file itself? 
If so, then what does `People with admin or owner permissions can set up a CODEOWNERS file in a repository` imply to?

**Comment by antonblr** [comment, marked as answer]
I think this means that only the admins/owners can select "Require review from Code Owners" (setup `CODEOWNERS` enforcement). 
Then anyone with `write` permissions is free to PR changes to that file with codeowners approvals, of course.

**Reply by ystndr** [reply]
@antonblr Thanks, if that's true, then the wording is quite confusing I guess

**Reply by antonblr** [reply]
You can always propose and PR clarifications into https://github.com/github/docs/blob/main/content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md 😏

**Reply by ystndr** [reply]
Also double-confirmed by GitHub support, exactly what you're saying

<img width="975" alt="image" src="https://user-images.githubusercontent.com/31244806/196573596-1b18e741-0798-4ca1-b016-f8d2ada409fc.png">

**Comment by ossanna16** [comment]
Hey there @ystndr and thank you for asking a good question! I'm glad you found our community 🙂
